packages/world/src/WorldContext.sol

```diff
@@ -1,6 +1,8 @@
 // SPDX-License-Identifier: MIT
 pragma solidity >=0.8.0;
 
+import { StoreSwitch } from "@latticexyz/store/src/StoreSwitch.sol";
+
 // Similar to https://eips.ethereum.org/EIPS/eip-2771, but any contract can be the trusted forwarder.
 // This should only be used for contracts without own storage, like Systems.
 contract WorldContext {
```

Comment:
should this be abstract?

Reply:
Can you elaborate?

Reply:
`abstract contract WorldContext` so it can't be instantiated on its own

Reply:
Oh i see, sure can't hurt